pkg/virt-config/config_test.go

```diff
@@ -482,4 +482,18 @@ var _ = Describe("ConfigMap", func() {
 		emulation = clusterConfig.IsUseEmulation()
 		Expect(emulation).To(BeFalse())
 	})
+
+	table.DescribeTable("when feature-gate", func(openFeatureGates string, isLiveMigrationEnabled, isSRIOVLiveMigrationEnabled bool) {
+		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.FeatureGatesKey: openFeatureGates},
+		})
+
+		Expect(clusterConfig.LiveMigrationEnabled()).To(Equal(isLiveMigrationEnabled))
+		Expect(clusterConfig.SRIOVLiveMigrationEnabled()).To(Equal(isSRIOVLiveMigrationEnabled))
+	},
+		table.Entry("LiveMigration and SRIOVLiveMigration are closed, both should be closed", "", false, false),
+		table.Entry("LiveMigration and SRIOVLiveMigration are open, both should be open", "LiveMigration,SRIOVLiveMigration", true, true),
+		table.Entry("SRIOVLiveMigration is open, LiveMigration should be open", "SRIOVLiveMigration", true, true),
```

Comment:
I was thinking about this for some time, but I have hard time to decide if it makes sense or not.
LiveMigration controls the operation for all cases, be it SR-IOV or anything else. The new one is more specific, only for SR-IOV, which makes it a but off that a specific one will control all implicitly.

@vladikr , what do you think about this case?